src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx

```diff
@@ -0,0 +1,65 @@
+import { t } from '@lingui/macro'
+import { Col, Form, Input, Row, Space } from 'antd'
+import { FormImageUploader } from 'components/inputs/FormImageUploader'
+import { CreateCollapse } from '../../CreateCollapse'
+
+// TODO: Do we want to set validators on these?
+export const ProjectDetailsPage: React.FC = () => {
+  return (
+    <Form name="projectDetails" colon={false} layout="vertical">
+      <Space direction="vertical" size="large" style={{ width: '100%' }}>
+        <Form.Item name="projectName" label={t`Project Name`}>
+          <Input placeholder={t`Enter project name`} />
```

Comment:
**suggestion** I didn't point it out in design but IMO we should avoid all placeholders. https://www.nngroup.com/articles/form-design-placeholders/